ethereum_clients/ControlledProcess.js

```diff
@@ -64,20 +64,18 @@ class ControlledProcess extends EventEmitter {
       })
 
       proc.on('close', code => {
-        if (code === 0) {
-          this.state = STATES.STOPPED
-          this.emit('newState', 'stopped')
-          this.debug('Emit newState: stopped')
-          return
+        this.state = STATES.STOPPED
```

Comment:
i am a bit skeptic that it worked well before. The behavior of arrow functions might be tricky using in combination with EventEmitter.

"It is possible to use ES6 Arrow Functions as listeners, however, when doing so, the `this` keyword will no longer reference the EventEmitter instance ([reference](https://nodejs.org/api/events.html#events_passing_arguments_and_this_to_listeners))."

here, `this` did not have the expected value.
![image](https://user-images.githubusercontent.com/47108/61716094-67041d80-ad2c-11e9-9d81-43959c49227c.png)